### PR TITLE
Unreal: launching on Linux

### PR DESCRIPTION
**Pull request by antirotor**
This is fixing issues of launching Unreal on Linux.

🔙 **Backport to 2.x** - #1673

### CODE REVIEW

**Comment by antirotor** [comment]
Depends on #1674

**Comment by antirotor** [comment]
This already includes #1679

File: openpype/hosts/unreal/api/lib.py [hunk 1]
```diff
@@ -180,32 +245,32 @@ def create_unreal_project(project_name: str,
     }
 
     if preset["install_unreal_python_engine"]:
-        # If `OPENPYPE_UNREAL_ENGINE_PYTHON_PLUGIN` is set, copy it from there
-        # to support offline installation.
+        # If `PYPE_UNREAL_ENGINE_PYTHON_PLUGIN` is set, copy it from there to
```
Comment (iLLiCiTiT) [inline openpype/hosts/unreal/api/lib.py:248]: Why key was changed `OPENPYPE_UNREAL_ENGINE_PYTHON_PLUGIN` -> `PYPE_UNREAL_ENGINE_PYTHON_PLUGIN`?

Comment (iLLiCiTiT) [reply openpype/hosts/unreal/api/lib.py:248]: BTW is it used only here? It doesn't seem that path set there is validated.

Comment (antirotor) [reply openpype/hosts/unreal/api/lib.py:248]: Actually this part is mainly deprecated. I'll mark it so and we'll remove it from future releases.

**Review by iLLiCiTiT — APPROVED** [review]
Tried only on Windows (it's not only linux changing) and it works. I have to trust that it works on linux as that needs to be compiled which I'm not capable of. It works from 4.26 on. We should mention that in docs if it's not there.

**Comment by iLLiCiTiT** [comment]
Can we add Unreal icon fix in this PR too?

**Comment by iLLiCiTiT** [comment]
NOTE:
We could maybe move the project preparation from prelaunch hook move to Python script which is executed as different process. So it's separated from tray and we may be able to display that "something is happening" with progress bar?

**Comment by antirotor** [comment]
> Tried only on Windows (it's not only linux changing) and it works. I have to trust that it works on linux as that needs to be compiled which I'm not capable of. It works from 4.26 on. We should mention that in docs if it's not there.

We should even stop from trying to run it with older versions.

> Can we add Unreal icon fix in this PR too?

Damn, I thought it is already in.

> We could maybe move the project preparation from prelaunch hook move to Python script which is executed as different process. So it's separated from tray and we may be able to display that "something is happening" with progress bar?

Possibly, but where would you display such progress bar?

**Comment by iLLiCiTiT** [comment]
> Possibly, but where would you display such progress bar?

It is just an idea for future feature. Because if you'll run python script you can use Qt in it (like we do in nonpython hosts). But that may also cause other issues.

**Pull request merged**